examples/all-erc20/README.md

```diff
@@ -1,5 +1,31 @@
-# all-erc20
+# Index all erc20 on the whole of Ethereum
 
 Example of using hypersync-client to get data for all erc20 transfers.
 
-Can run `npx tsc && node dist/app.js` to run it.
```

Comment:
![image](https://github.com/enviodev/hypersync-client-node/assets/6032276/17d8f7f8-e862-40bb-9f24-af62028b3d8f)


I think it works with `ts-node`. Maybe we should move to that rather: https://www.npmjs.com/package/ts-node